### DISCUSSION TITLE
Oracle DateTime loaded as Date using ConnectorX

### DISCUSSION THREAD

**Discussion by mpenner77** [sfu-db/connector-x#328] [Q&A]
I've been using Pandas and SqlAlchemy to get data from a Oracle database and save it as Parquet files for analysis.

I've read that Connectorx is the fastest and most memory efficient way to load data from databases into Python and I want to try it.

My routine with sqlalchemy takes about 6 min to get 2M rowx x 8 columns from Oracle DB. ConnectorX takes 1.15 min to get the same data. It was quite impressive.

But, when I checked my files generate with ConnectorX the datetime column was loaded as only date.

Is there way to get datetime from ConnectorX using a Oracle Db?

**SqlAlchemy** 
694882 TESTE CPD 4 **2020-08-31 17:33:53** 99999
694888 TESTE CPD 10 **2020-08-31 16:10:30** 99999
694888 TESTE CPD 10 **2020-08-31 16:08:19** 99999

**ConnectorX** 
694882 TESTE CPD 4 **2020-08-31** 99999
694888 TESTE CPD 10 **2020-08-31** 99999
694888 TESTE CPD 10 **2020-08-31** 99999

**Comment by wangxiaoying** [comment]
Hi @mpenner77 , may I ask what's the schema of your "datetime column"? 

According to oracle's documentation, the "datetime_datatype" includes `date`, `timestamp` and `interval`. We support both `date` time `timestamp`. And in our test the `TEST_TIMESTAMP` column works fine with both date and time:

<img width="1863" alt="image" src="https://user-images.githubusercontent.com/5569610/185513720-86c9fe76-2bfc-4c21-bba5-eb937a454fa1.png">

Can you provide the corresponding SQL to create your table, so we could reproduce your error?
